projects/memapCore/src/main/java/memap/components/prototypes/MEMAPCoordination.java

```diff
@@ -68,6 +69,10 @@ public MEMAPCoordination(TopologyController topologyController) {
 
 	@Override
 	public void handleRequest() {
+		if(optResult.resultMap.get("CoSESEMS01OBJ01_CPROD01") != null) {
+			double producerSetpoint = optResult.resultMap.get("CoSESEMS01OBJ01_CPROD01")[0];
+			System.out.println("Producer Setpoint at Start: " + producerSetpoint);
+		}
```

Comment:
Another check is needed probably. If in "Plannig Tool" mode, we do not have this value at all.

Reply:
Absolutely. I mentioned that "a lot of loging is still included to check the values. **These can be removed after review and before merging.**"